code/index.html

```diff
@@ -4,19 +4,75 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Urbanist:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald&display=swap" rel="stylesheet">
     <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
   <body>
-    <header>
+    <header class="header">
       <!-- Some kind of logo and navbar -->
+      <img class="logo" src="./logo.PNG" alt="The View Report"/>
+      <nav class="navigation">
+        <a href="#">Search</a>
+        <a href="#">Menu</a>
+        <a href="#">Login</a>
+        <a href="#">Subscribe</a>
+        <div class="burger">
+          <span class="line"></span>
+          <span class="line"></span>
+          <span class="line"></span>
+        </div>
+      </nav>
     </header>
 
-    <section class="big-news">
+    <article class="big-news-section">
       <!-- The big news content -->
-    </section>
+      <img class="tortoise" src="./Tortoise.jpg" alt="Tortoise"/>
+      <p class="big-news-paragraph"><span style="font-weight:bold">Horrifying and Amazing: Giant Tortoise Attacking and Eating Baby Bird.</span></br>
+        Chase in Seychelles is first known example of hunting in the wild by creature thought to be herbivore. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+      </article>
 
-    <section class="">
```

Comment:
This class doesn't have any name. You may explore adding one or removing the section.